mdata/store_cassandra.go

```diff
@@ -362,7 +366,12 @@ func (o asc) Less(i, j int) bool { return o[i].sortKey < o[j].sortKey }
 
 func (c *CassandraStore) processReadQueue() {
 	for crr := range c.readQueue {
-		cassGetWaitDuration.Value(time.Since(crr.timestamp))
+		waitDuration := time.Since(crr.timestamp)
+		cassGetWaitDuration.Value(waitDuration)
+		if waitDuration > c.omitReadTimeout {
```

Comment:
pushed another commit for that https://github.com/raintank/metrictank/pull/685/commits/e5b7653cbdbb949db36917b5f31699c2edf75231